libs/blocks/section-metadata/section-metadata.css

```diff
@@ -134,15 +134,30 @@ main>.section[class*='-up']>.content {
 .section.four-up,
 .section.five-up {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(276px, 1fr));
+  grid-template-columns: 1fr;
   gap: var(--spacing-m);
   align-items: start;
   padding-left: var(--grid-margins-width);
   padding-right: var(--grid-margins-width);
 }
 
-.section.five-up {
-  grid-template-columns: repeat(auto-fit, minmax(142px, 1fr));
+@media screen and (min-width:900px){
+  .section.two-up,
+  .section.three-up,
+  .section.four-up {
+    grid-template-columns: repeat(auto-fit, minmax(234px, 1fr));
+  }
+
+  .section.five-up {
+    grid-template-columns: repeat(auto-fit, minmax(142px, 1fr));
+  }
+}
+
+@media (max-width: 600px) {
+  .section[class*='-up'] .mobile-max-width {
```

Comment:
```suggestion
  .section[class*='-up'] .max-width-mobile {
```

Reply:
Nit: we have other `-mobile` css names. Probs wanna stick w/ the same convention. Thanks

Reply:
Where are there imports using `mobile-max-width` - I don't see any? I really think keeping to our naming convention with the media query at the end is best. @rgclayton  - Any thoughts?

Reply:
I'm working on one and there's one for international pages. I'm confused, I thought we were good with `mobile max width`? https://adobe-mwp.slack.com/archives/C03B0BUA75E/p1683150420228059?thread_ts=1682536029.448519&cid=C03B0BUA75E

Reply:
I needed a chance to review this and I think following a naming convention is necessary. Sorry this is nit picky, but once its in, it's cemented. Lets fix it while we can